_targets.R

```diff
@@ -6,10 +6,44 @@ source("3_visualize/src/plot_timeseries.R")
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+p_width <- 12
+p_height <- 7
+p_units <- "in"
 p1_targets_list <- list(
+  tar_target(
+    site_data_01427207_csv,
+    download_nwis_site_data(filepath =  "1_fetch/out/nwis_01427207_data.csv"),
```

Comment:
Nice work including both the file extension in the target name and the filepath in the function call. 👍